sites/skeleton.dev/src/content/docs/guides/cookbook/light-switch.mdx

```diff
@@ -0,0 +1,64 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+title: LightSwitch
```

Comment:
We need to ensure consistent usage for the name. I think we're mixing between the following in a few places:

- Light Switch
- LightSwitch
- Lightswitch

Specifically between the cookbook URL, the Cookbook page, and the Dark Mode doc page.

![Screenshot 2025-03-24 at 12 59 50 PM](https://github.com/user-attachments/assets/2b5a17b0-b344-413a-b452-bfdec561204c)

Wikipedia says it's two words:
https://en.wikipedia.org/wiki/Light_switch